docs/docs/configuration/providers/openid_connect.md

```diff
@@ -20,15 +20,15 @@ To configure the OIDC provider for Dex, perform the following steps:
 
    See the [getting started guide](https://dexidp.io/docs/getting-started/) for more details.
 
-2. Setup oauth2-proxy with the correct provider and using the default ports and callbacks. Add a configuration block to 
+2. Setup oauth2-proxy with the correct provider and using the default ports and callbacks. Add a configuration block to
    the `staticClients` section of `examples/config-dev.yaml`:
 
-    ```
+    ```yaml
     - id: oauth2-proxy
     redirectURIs:
     - 'http://127.0.0.1:4180/oauth2/callback'
     name: 'oauth2-proxy'
-    secret: proxy
+    public: true
```

Comment:
This shouldn't be replaced. People are still going to use the client secrets and will require the documentation for this. Therefore we need to add an additional section that explain the differences and how to properly configure PKCE